app/views/sessions/new.html.erb

```diff
@@ -1,18 +1,28 @@
-<% @hide_session_info = true %>
+<%= image_tag('openstax-logo-white.png', width: 174, height: 40, class: 'logo') %>
```

Comment:
Yes, the second screen with password/social choices should also have it.  I've extracted into a partial for use on both screens.